Metagenomics/tutorials/mothur-miseq-sop.md

```diff
@@ -0,0 +1,1516 @@
+
+Now you may say, "But wait I want that stuff". Fine. But, the primers we use, are only supposed to amplify members of the Bacteria and if they're hitting Eukaryota or Archaea, then it is a mistake. Also, realize that chloroplasts and mitochondria have no functional role in a microbial community.
+
+Let's go ahead and classify those sequences using the Bayesian classifier with the `classify.seqs` command:
+
+> ### :pencil2: Hands-on: Remove undesired sequences
+> - **Tool:** Classify.seqs
+> - **Parameters:**
+>   - Set **fasta** to the fasta output from Remove.seqs
+>   - Set **reference** to `trainset9032012.pds.fasta` from your history
+>   - Set **taxonomy** to `trainset9032012.pds.tax` from your history
+>   - Set **count** to the count table file from Chimera.uchime
+>   - Set **cutoff** to 80
+>
+> Have a look at the taxonomy output. You will see that every read now has a classification.
+>
+> Now that everything is classified we want to remove our undesirables. We do this with the remove.lineage
+> command:
+>
+> - **Tool:** Remove.lineage
+> - **Parameters:**
+>   - Set **taxonomy** to the taxonomy output from Classify.seqs
+>   - Set **taxon** to `Chloroplast-Mitochondria-unknown-Archaea-Eukaryota` in the textbox under *Manually
+> select taxons for filtering*
+>   - Set **fasta** to the fasta output from Remove.seqs
+>   - Set **count** to count table from Chimera.uchime
+{: .hands_on}
+
+Also of note is that *unknown* only pops up as a classification if the classifier cannot classify your sequence to one of the domains.
+
+> ### :question: Questions
+>
+> 1. How many unique (representative) sequences were removed in this step?
+> 2. How many sequences in total?
+>
+>    <details>
+>      <summary> Click to view answer</summary><br>
+>      20 representative sequences were removed. <br>
+>      The fasta file output from Remove.seqs had 2628 sequences while the fasta output from Remove.lineages
+>      contained 2608 sequences.
+>      <br><br>
+>      162 total sequences were removed. <br>
+>      If you run summary.seqs with the count table, you will see that we now have 2608 unique sequences
+>      representing a total of 119,168 total sequences (down from 119,330 before). This means 162 of our  
+>      sequences were in represented by these 20 representative sequences.
+>    </details>
+{: .question}
+
+At this point we have curated our data as far as possible and we're ready to see what our error rate is.
+
+
+## Assessing error rates based on our mock community
+
+Measuring the error rate of your sequences is something you can only do if you have co-sequenced a mock community. This is something we include for every 95 samples we sequence. You should too because it will help you gauge your error rates and allow you to see how well your curation is going, and whether something is wrong with your sequencing setup.
```

Comment:
It could be nice to explain what is a mock community and how to obtain it. What do you think?